llmcam/oas_to_requests.py

```diff
@@ -177,10 +177,51 @@ def transform_property(
     return {k: v for k, v in new_prop.items() if v is not None}, required
 
 
-# %% ../nbs/04_oas_to_requests.ipynb 21
+# %% ../nbs/04_oas_to_requests.ipynb 22
+def generate_request(
+    function_name: str,  # The name of the function
+    url: str,  # The URL of the request
+    method: str,  # The method of the request
+    path: dict = {},  # The path parameters of the request
+    query: dict = {},  # The query parameters of the request
+    body: dict = {},  # The body of the request
+    accepted_queries: list = [],  # The accepted queries of the request
+    **kwargs  # Additional parameters
+) -> dict:  # The response of the request
+    """Generate a request from the function name and parameters."""
+    # Extract the URL and method from the toolbox if not provided
+    if url is None or method is None:
+        raise ValueError("URL and method must be provided.")
+    
+    # Prepare the request
+    headers = {
+        "Content-Type": "application/json"
+    }
+
+    # Extract the accepted queries
+    queries = {k: v for k, v in query.items() if k in accepted_queries}
+    queries.update({k: v for k, v in kwargs.items() if k in accepted_queries})
+
+    # Execute the request
+    response = requests.request(
+        method,
+        url.format(**path, **kwargs),
+        headers=headers,
+        params=queries if len(queries) > 0 else None,
+        json=body if len(body) > 0 else None
+    )
+
+    # Return the response (either as JSON or text)
+    try:
+        return response.json()
+    except:
+        return {"message": response.text}
+
+# %% ../nbs/04_oas_to_requests.ipynb 27
 def toolbox_schema(
```

Comment:
The name of `toolbox_schema` is confusing. Actually this is `oas_schema`, right?